mono/mini/mini-wasm.h

```diff
@@ -34,6 +35,7 @@
 //mini-codegen stubs - this doesn't do anything
 #define MONO_ARCH_CALLEE_REGS (1 << 0)
 #define MONO_ARCH_CALLEE_FREGS (1 << 1)
+#define MONO_ARCH_CALLEE_XREGS (1 << 1)
```

Comment:
Should this be 1 << 2 with the others adjusted?

Also, this group of MONO_ARCH defines appears to be duplicated by another group below it, which is a hazard. We should get rid of one of the duplicates

Reply:
Fixed.